Make the tally transmitter more visually appealing

zusorio opened the pull request:
- I reused and slightly modified the UI from the tally viewer
- I also removed an unused parameter
![image](https://user-images.githubusercontent.com/23165606/213927901-20f97e5e-cda5-4606-9a7f-6e8fa9f50d97.png)
![image](https://user-images.githubusercontent.com/23165606/213927906-56dc56ea-de17-4e3f-bac4-418e378682f6.png)

slmnio commented on website/src/components/broadcast/roots/WebsocketTransmitter.vue:
```diff
@@ -1,20 +1,34 @@
+            <i class="fas fa-check-circle"></i> Connected
+        </div>
+        <div v-else>
+            <LoadingIcon/>
```
I think this should probably be a disconnected icon rather than loading icon - realistically with the websocket connection all being local we're not even going to see a loading stage, so it's either going to be connected or not.

slmnio commented on website/src/components/broadcast/roots/WebsocketTransmitter.vue:
```diff
@@ -1,20 +1,34 @@
+            <i class="fas fa-check-circle"></i> Connected
+        </div>
+        <div v-else>
+            <LoadingIcon/>
```
I think this should probably be a disconnected icon rather than loading icon - realistically with the websocket connection all being local we're not even going to see a loading stage, so it's either going to be connected or not.

slmnio replied:
Additionally - it's likely that a disconnected socket is going to be erroring - can we display that error?

zusorio replied:
This should be addressed in c63a1fe251c72167216e7145e486b9489504f1e8

slmnio commented on website/src/components/broadcast/roots/WebsocketTransmitter.vue:
```diff
@@ -112,7 +119,54 @@ export default {
+    text-align: center;
+}
+
+.prod-scenes .prod-preview {
+    color: lime;
+    border-color: lime;
+    border-radius: .1em;
+}
+
+.prod-scenes .prod-program {
+    color: #ff4646;
+    border-color: #ff0000;
+    border-radius: .1em;
+}
```
I wonder whether this should be in a more global scope, it's used on a few parts of tally data, and on the dashboard now. Bit out of scope of this PR but whatever

zusorio replied:
Yeah I agree, should probably be a seperate PR though.
This usage of it is also slightly customized to work better at the size it's at so we'd have to account for that.

slmnio reviewed (changes requested):
I really like the design, I think it improves it a lot. Few notes on the disconnected state that we talked about before.

zusorio commented:
I added error messages and changed the icon in c63a1fe251c72167216e7145e486b9489504f1e8, here's what it looks like
![image](https://user-images.githubusercontent.com/23165606/214051140-5677dd07-2fa6-425e-b64f-208292da5606.png)

slmnio reviewed (approved):
All changes look great!
